manifest/common_test.go

```diff
@@ -103,15 +104,23 @@ func TestCompressionVariantMIMEType(t *testing.T) {
 		algo     *compressiontypes.Algorithm
 		expected string
 	}{
-		{"AU", nil, "AU"}, {"AU", &compression.Gzip, "AG"}, {"AU", &compression.Zstd, ""},
-		{"AG", nil, "AU"}, {"AG", &compression.Gzip, "AG"}, {"AG", &compression.Zstd, ""},
+		{"AU", nil, "AU"},
+		{"AU", &compression.Gzip, "AG"},
+		{"AU", &compression.Zstd, ""},
+		{"AG", nil, "AU"},
+		{"AG", &compression.Gzip, "AG"},
+		{"AG", &compression.Zstd, ""},
 		{"BU", &compression.Zstd, ""},
 		{"BG", &compression.Zstd, ""},
-		{"CG", nil, ""}, {"CG", &compression.Zstd, "CZ"},
-		{"CZ", nil, ""}, {"CZ", &compression.Gzip, "CG"},
+		{"CG", nil, ""},
+		{"CG", &compression.Zstd, "CZ"},
+		{"CZ", nil, ""},
+		{"CZ", &compression.Gzip, "CG"},
 		{"DG", nil, ""},
-		{"unknown", nil, ""}, {"unknown", &compression.Gzip, ""},
-		{"", nil, ""}, {"", &compression.Gzip, ""},
+		{"unknown", nil, ""},
+		{"unknown", &compression.Gzip, ""},
+		{"", nil, ""},
+		{"", &compression.Gzip, ""},
 	} {
```

Comment:
These are currently organized one line per input “MIME type”, e.g. all for `AU` on a single line.

I suppose the test could be restructured to make that an explicit data structure instead; with extra type names that might end up with more noise, I’m not sure.